firstvoices/backend/models/signals/visibility_signals.py

```diff
@@ -0,0 +1,27 @@
+from django.db import transaction
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from backend.models.jobs import BulkVisibilityJob, JobStatus
+from backend.tasks.visibility_tasks import bulk_visibility_change_job
+from firstvoices.celery import link_error_handler
+
+
+@receiver(post_save, sender=BulkVisibilityJob)
+def request_bulk_visibility_change(sender, instance, **kwargs):
+    """
```

Comment:
Is a signal necessary here? Since we only want to start this job one time, it seems like it might be easier to just queue the job in a regular function call after saving the initial job instance.